internal/servers/controller/handlers/health/health_test.go

```diff
@@ -0,0 +1,88 @@
+package health
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/boundary/internal/gen/controller/ops/services"
+	"github.com/hashicorp/boundary/sdk/pbs/controller/ops/resources/health"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+func TestGetHealth(t *testing.T) {
```

Comment:
This appears to only test the grpc response.  Consider writing a test against the http server similar to the TestAuthenticationHandler in handler_test.go.  That way you can confirm http status codes along with the response body.

Reply:
Thanks for the comment! Yes, this only tests the gRPC side of things. There will be some middleware that I will be writing to set the correct HTTP status code based on the gPRC message, but that's not in yet.

Reply:
Added in [6fad34e](https://github.com/hashicorp/boundary/pull/1882/commits/6fad34e659e7280de1dfad9be839f6e562447c1a)